FileConv/Gfx/Fontrix-notes.md

```diff
@@ -0,0 +1,51 @@
+# Fontrix Fonts #
+
+File types:
+
+- BIN / $6400: Fontrix Font File
+
+Primary references:
+
+- Reverse engineering
+
+## Fonts ##
+
+Each font contains between 1 and 94 characters.
+
+Character data is stored one bit per pixel, where 0 indicated background and 1 is foreground. Data
+for each glyph is stored in byte order from left to right, top to bottom,  with the left-most pixel
+of each byte stored in the LSB.  Glyph data is referenced by a pointer into the file for each
+character.  Fonts need not contain all characters; there are values identifying  the first character
+and number of characters included in the file.  Regardless of how many characters are included,
+there are always 96 data pointers, representing ASCII values 32 through 127, although the first and
+last characters are never defined nor used by Fontrix. Although the is not necessarily glyph data
+provided for it, the offset for character 127 always points to the first byte beyond the  glyph data
+for character 126.
+
+The glyph data is always mapped sequentially
+and the width in bytes of a character can be determined by the difference between its data offset
+and that of the following character, divided by the given glyph height.
+
+Offsets $0E2-$141 appear to potentially be the individual character widths for proportional fonts,
+although further research needs to be done to confirm this.
+
+The file format is:
+
+```
++$000/001: - Unknown byte
```

Comment:
Can remove the " - " on these entries.

I've been omitting the leading zeroes on the field lengths (1 vs. 001) because I think it's a little easier to read.  It feels reasonable on the hex offsets, but not for a simple count.  I doubt I'm entirely consistent though.  I usually have to stop and look at another doc to remember what I've been doing, and sometimes I do something lumpy like https://github.com/fadden/CiderPress2/blob/main/DiskArc/Disk/Nibble-notes.md#525-floppy-disks .

Reply:
Done.  This'll be in the next update.